guest-tools/regular_domain.xml.template

```diff
@@ -0,0 +1,60 @@
+<domain type='kvm' xmlns:qemu='http://libvirt.org/schemas/domain/qemu/1.0'>
```

Comment:
I think we agreed to rename this file `regular_vm.xml.template`.

Reply:
Ok, missed that